NEWS.md

```diff
@@ -1,3 +1,7 @@
+# admiral development version
```

Comment:
Why did you use "development version" instead of "0.6.0"?

It seems that it prevents the section to be displayed. Is this intended?

Reply:
I don't think it's set in stone that the next version will be 0.6.0 so I used "development version" instead. That shouldn't affect the rendering, though. But even if it does, this will obviously be replaced by an actual version number in the next release (and only then will it be displayed on our website).